docs/development_guide.md

```diff
@@ -51,7 +55,62 @@ Note that after running the above command we get some dependency conflicts in
 installed packages which is currently safe to ignore. For details see
 [Issue #71](https://github.com/googlegenomics/gcp-variant-transforms/issues/71).
 
+### Setup IDE
+
+You may choose any IDE as you like. The following steps are intended for
+IntelliJ users.
+
+#### Install IntelliJ IDE
+
+Download
+[IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=linux)
+and install.
+
+#### Install Python plugin
+
+Choose File | Settings on the main menu, and then go to Plugins.
+Click the Install JetBrains plugin button.
+In the dialog that opens, search for Python Community Edition and then install
+the plugin.
+
+For more details, refer to
+[Install plugins](https://www.jetbrains.com/help/idea/installing-updating-and-uninstalling-repository-plugins.html).
+
+#### Create Project
+
+Choose File | New | Project on the main menu, and create a new Python project
+in the dialog that opens. To setup the Project SDK, follow the following steps.
```

Comment:
Please add a step before this which says where Project SDK is, i.e., from File | Project Structure go to the Project sub-menu ...

Reply:
That is one way to setup the SDK. Also, we can setup it directly in the new project dialog.

Reply:
Oh, I see.